profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.info

```diff
@@ -0,0 +1,260 @@
+name = Nexteuropa Newsroom
+description = Integration with the Newsroom application
+core = 7.x
+package = NextEuropa
+// @codingStandardsIgnoreLine
+version = 7.x-2.0
+// @codingStandardsIgnoreLine
+project = nexteuropa_newsroom
+dependencies[] = entity_translation
+dependencies[] = geofield
+dependencies[] = feeds_et
+dependencies[] = feeds
+dependencies[] = feeds_tamper
+dependencies[] = feeds_xpathparser
+dependencies[] = i18n_taxonomy
+dependencies[] = title
+dependencies[] = views_tree
+configure = admin/config/content/newsroom
+feature_sets[nexteuropa_newsroom][title] = Nexteuropa newsroom
+feature_sets[nexteuropa_newsroom][description] = Use the nexteuropa newsroom to integrate your website with the corporate news service provided by DG connect, it will provide blocks with contextual news items selected basing on your own configuration of the newsroom. <a class="description" href="https://webgate.ec.europa.eu/fpfis/wikis/display/MULTISITE/%5BFPFIS%5D+Nexteuropa+Newsroom+-+Documentation">Read documentation</a>
+feature_sets[nexteuropa_newsroom][icon] = nexteuropa_newsroom.png
+feature_sets[nexteuropa_newsroom][enable][] = nexteuropa_newsroom
+feature_sets[nexteuropa_newsroom][disable][] = nexteuropa_newsroom
+feature_sets[nexteuropa_newsroom][uninstall][] = nexteuropa_newsroom
+feature_sets[nexteuropa_newsroom][category] = Communication
+features[context][] = newsroom
+features[context][] = newsroom_item
+features[context][] = newsroom_site_wide
+features[ctools][] = context:context:3
+features[ctools][] = feeds:feeds_importer_default:1
+features[ctools][] = feeds_tamper:feeds_tamper_default:2
+features[ctools][] = field_group:field_group:1
+features[ctools][] = strongarm:strongarm:1
+features[ctools][] = views:views_default:3.0
+features[features_api][] = api:2
+features[feeds_importer][] = newsroom_deleted_items
+features[feeds_importer][] = newsroom_unpublished_items
+features[feeds_tamper][] = newsroom_unpublished_items-blank_source_1-unpublish_node
+features[field_base][] = field_associated_newsroom_select
```

Comment:
Please rename those fields what don't have the 'newsroom' part to make them enough unique otherwise in the platform those can be in conflict when the feature will be enabled.